src/System.Private.ServiceModel/tests/Common/Scenarios/Endpoints.cs

```diff
@@ -9,191 +9,41 @@
 
 public static partial class Endpoints
 {
-    private static X509Certificate2 rootCert;
```

Comment:
Reviewers: as mentioned elsewhere, this code moved verbatim into ServiceUtilHelper.cs